tests/test_ideogram.py

```diff
@@ -0,0 +1,359 @@
+    assert num_chromosoms == 46
+
+
+def test_chromosomes(dash_threaded, selenium):
+    """A list of the numbers of the chromosomes to display."""
+
+    prop_type = 'list'
+
+    def assert_callback(prop_value, nclicks, input_value):
+        answer = ''
+        if nclicks is not None:
+            answer = FAIL
+            if PROP_TYPES[prop_type](input_value) == prop_value:
+                answer = PASS
+        return answer
+
+    template_test_component(
+        dash_threaded,
+        selenium,
+        APP_NAME,
+        assert_callback,
+        ideogram_test_props_callback,
+        "chromosomes",
+        "1,3,17",
+        prop_type=prop_type,
+        component_base=COMPONENT_REACT_BASE,
+        **BASIC_PROPS
+    )
+
+    btn = wait_for_element_by_css_selector(selenium, '#test-{}-btn'.format(APP_NAME))
+    btn.click()
+
+    num_chromosoms = len(selenium.find_elements_by_class_name('chromosome-set-container'))
+    assert num_chromosoms == 3
+
+
+def test_chromosomes_wrong_input(dash_threaded, selenium):
+    """A list of the numbers of the chromosomes to display."""
+
+    prop_type = 'list'
+
+    def assert_callback(prop_value, nclicks, input_value):
+        answer = ''
+        if nclicks is not None:
+            answer = FAIL
+            if PROP_TYPES[prop_type](input_value) == prop_value:
+                answer = PASS
+        return answer
+
+    template_test_component(
+        dash_threaded,
+        selenium,
+        APP_NAME,
+        assert_callback,
+        ideogram_test_props_callback,
+        "chromosomes",
+        "1,D,3",
+        prop_type=prop_type,
+        component_base=COMPONENT_REACT_BASE,
+        **BASIC_PROPS
+    )
+
+    btn = wait_for_element_by_css_selector(selenium, '#test-{}-btn'.format(APP_NAME))
+    btn.click()
+
+    num_chromosoms = len(selenium.find_elements_by_class_name('chromosome-set-container'))
+    assert num_chromosoms == 2
+
+
+def test_brush(dash_threaded, selenium):
+    """Genomic coordinate range (e.g. "chr1:104325484-119977655") for a brush on a chromosome."""
+
+    prop_type = 'str'
+
+    def assert_callback(prop_value, nclicks, input_value):
+        answer = ''
+        if nclicks is not None:
+            answer = FAIL
+            if PROP_TYPES[prop_type](input_value) == prop_value:
+                answer = PASS
+        return answer
+
+    template_test_component(
+        dash_threaded,
+        selenium,
+        APP_NAME,
+        assert_callback,
+        ideogram_test_props_callback,
+        "brush",
+        "chr3:3500000-40000000",
+        prop_type=prop_type,
+        component_base=COMPONENT_REACT_BASE,
+        chromosomes=['3'],
+        orientation="horizontal",
+        brush="chr3:200000-2000000",
+        **BASIC_PROPS
+    )
+
+    # verify the existence of the brush
+    brush = selenium.find_elements_by_class_name('brush')
+    assert len(brush) == 1
+    # selection = selenium.find_elements_by_class_name('selection')[0]
```

Comment:
This test currently fails (issue #186), it has been commented for the time being